roda-ui/roda-wui/src/main/resources/config/theme/Banner.html

```diff
@@ -1,2 +1,3 @@
 <img title="RODA" class="homeLogo" alt="RODA logo" src="api/v1/theme?resource_id=roda2-logo.png"/>
-<div class="homeText">RODA COMMUNITY EDITION</div>
+<div class="homeText">RODA 5 <span class="homeSubText">COMMUNITY EDITION</span></div>
```

Comment:
You need to change the other languages too

Reply:
done